cypress/integration/specialFeatures/mediaPlayerPreRollAds/index.js

```diff
@@ -0,0 +1,49 @@
+import visitPage from '../../../support/helpers/visitPage';
+import getAppEnv from '../../../support/helpers/getAppEnv';
+import appToggles from '../../../support/helpers/useAppToggles';
+
+const pageType = 'mediaAssetPage';
+
+const embedUrlConfig = {
+  test: [
+    'https://polling.test.bbc.co.uk/ws/av-embeds/cps/bengali/media-23269006/p01k6msp/bn', // CPS Video Clip
+    'https://polling.test.bbc.co.uk/ws/av-embeds/cps/arabic/world-23278971/p01m7d09/ar', // CPS Audio Clip
+    'https://polling.test.bbc.co.uk/ws/av-embeds/cps/swahili/media-23268999/bbc_one_northern_ireland_hd/sw', // CPS Live Stream
+  ],
+  live: [
+    'https://polling.bbc.co.uk/ws/av-embeds/cps/bengali/news-51660521/p085h0z3/bn', // CPS Video Clip
+    // CPS Audio Clip?
+    // TC2 Video Clip?
+    // TC2 Audio Clip?
+    // Live Stream?
+  ],
+};
+
+describe(`Tests for Preroll Ads for Media Player Embed`, () => {
+  // Get correct URLs based on application environment
+  const embedUrls = embedUrlConfig[getAppEnv()];
+
+  if (embedUrls) {
+    ['Canonical', 'AMP'].forEach(platform => {
+      const pathSuffix = platform === 'AMP' ? '/amp' : '';
+
+      embedUrls
+        .map(embedUrl => `${embedUrl}${pathSuffix}`)
+        .forEach(embedUrl => {
+          if (appToggles.prerollAds.enabled) {
+            it(`${platform} should load preroll ad plugin`, () => {
+              visitPage(`${embedUrl}?ads-debug`, pageType);
+
+              cy.get(`script[src*="dotcom-preroll.js"]`).should('exist');
+            });
+          }
+
+          it(`${platform} should not load preroll ad plugin`, () => {
+            visitPage(`${embedUrl}`, pageType);
+
+            cy.get(`script[src*="dotcom-preroll.js"]`).should('not.exist');
+          });
+        });
+    });
+  }
```

Comment:
Could we maybe log something in Cypress if embed urls returns false (i.e. when on a local environment?)?

Currently if ran on local env you just get an error: `“No tests found in your file:” for the specialFeatures/mediaPlayerPreRollAds/index.js`

It may be nicer to log something out in a test, something like:

```
it('will not run tests when on local environment', () => {
 cy.log(`There is no local embed URL, so this test suite does not run while in the local environment`);
})
```

Happy to approve but this would be a future dev-experience improvement.

Reply:
I'll raise an issue to improve this. Just need to get these tests in ASAP now!